src/router/routes.js

```diff
@@ -131,13 +131,6 @@ module.exports = [
 			excludeFromStaticSitemap: true,
 		}
 	},
-	{
-		path: '/funded/:id',
```

Comment:
Excited to see this removed an incorporated into the new BP!!!

We'll eventually need to clean up the [redirect in legacy](https://github.com/kiva/kiva/blob/development/sites/www_kiva/server/views/Xb/Loan/LoanView.php#L151) but can probably just turn off the Experiment (`API_Experiment_FundedProfileRedirect`) if we haven't already...